sdk/appconfiguration/app-configuration/test/public/featureFlag.spec.ts

```diff
@@ -191,11 +199,15 @@ describe("AppConfigurationClient - FeatureFlag", () => {
     let recorder: Recorder;
     let featureFlag: ConfigurationSetting<FeatureFlagValue>;
     beforeEach(async function (this: Context) {
-      recorder = startRecorder(this);
-      client = createAppConfigurationClientForTests() || this.skip();
+      recorder = new Recorder(this.currentTest);
+      await recorder.start(recorderStartOptions);
+      client =
+        createAppConfigurationClientForTests(recorder.configureClientOptionsCoreV1({})) ||
+        this.skip();
+      recorder.variable("name-1", `${getRandomNumber()}`);
```

Comment:
Not sure if the recorder is wired up for this, but something like `const name = recorder.variable("name-1", getRandomNumber().toString());` seems a little more natural to me. Does `recorder.variable` return the stored value if an initializer is passed?

Reply:
It does return